package.json

```diff
@@ -25,15 +25,20 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "jsonwebtoken": "^8.5.1",
+    "ms": "^2.1.2",
     "passport": "0.4.0",
     "passport-local": "1.0.0",
     "semver": "^6.3.0"
   },
   "devDependencies": {
-    "eslint": "^6.7.2",
+    "debug": "^4.1.1",
```

Comment:
the plugin will fail to execute if `NODE_ENV=production` because this dependency is loaded as-is by `index.js` without a guard preventing an exception to be thrown if the module isn't there

Reply:
assertion added